scipy/optimize/_tnc.py

```diff
@@ -312,7 +314,8 @@ def _minimize_tnc(fun, x0, args=(), jac=None, bounds=None,
         max(1,min(50,n/2)). Defaults to -1.
     maxiter : int, optional
         Maximum number of function evaluations. This keyword is deprecated
-        in favor of `maxfun`. Only if `maxfun` is None is this keyword used.
+        in favor of `maxfun` and will removed in SciPy 1.11.0.
+        Default is None.
```

Comment:
Sorry gh wont let me add a suggestion because of the deleted line in the middle and should have put this with the other comment but personally I would formatted this like
```
maxiter : int, optional, deprecated
    Maximum number of function evaluations, defaults to None.
    Only if `maxfun` is None is this keyword used.

    .. deprecated:: 1.9.0

    The keyword `maxiter` is deprecated in favor of `maxfun`.
    and will removed in SciPy 1.11.0.
```

Reply:
CI is already green and we are not consistent with this tag (for good or bad). Also I preferred to remove `Only if maxfun is None is this keyword used.` as it reads strange to me.